packages/e2e/features/ui/components/authenticator/sign-up-with-username.feature

```diff
@@ -9,10 +9,6 @@ Feature: Sign Up with Username
   Scenario: Login mechanism set to "username"
     Then I see "Username" as an input field
 
-  @angular @react @vue 
-  Scenario: "Preferred Username" is included from `aws_cognito_signup_attributes`
-    Then I see "Preferred Username" as a "text" field
-
```

Comment:
Since the `auth-with-username-no-attributes` environment does not include a `Preferred Username` attribute, this test is failing after the switch to said environment. I believe this test can be safely removed:
- Additional sign up attributes aren't necessarily core to the `sign-up-with-username` use cases
- We already test additional sign up attributes in the dedicated `sign-up-with-attributes`, which includes the `Preferred Username` attribute

Reply:
Makes sense to me, thanks for fixing this!